src/components/Select/src/SelectControl.vue

```diff
@@ -169,9 +169,9 @@ export default {
 	position: relative;
 	box-sizing: border-box;
 	min-width: 80px;
+	font-weight: var(--font-weight-paragraph, 400);
```

Comment:
There's several components that have an explicit style set for the font-weight and font-family that points to the paragraph -- but these styles seem unnecessary. 

When not using the theme component, every component will inherit whatever font style and weight are set at the top level or the browser default.

When using the theme component, every component will also inherit that as the default since those styles are set on the parent.

So I think we can remove these explicit styles that currently pointing to the font-family / weight for the paragraph since without them it should work exactly the same way.

Reply:
This is true assuming that Maker is being used to build an entire standalone app. This isn't true in the case of Website which is a large codebase which has a lot of Maker and non-Maker code intermixed with custom CSS overrides everywhere in between. It's better to code defensively and add redundant CSS styles in this scenario.

Reply:
I understand your perspective, but where does defensive CSS end? Shouldn't we trust consumers to make decisions about their styles without increasing the weight of our package? Wouldn't this be a good opportunity for Website to streamline / remove styles they don't need as well?

Reply:
i updated the pr to remove the explicit defensive css styles where possible